includes/class-wc-gateway-ppec-settings.php

```diff
@@ -303,8 +353,14 @@ public function currency_has_decimal_restriction() {
 	 */
 	public function get_paypal_locale() {
 		$locale = get_locale();
+
 		if ( ! in_array( $locale, $this->_supported_locales ) ) {
-			$locale = 'en_US';
+			// Mapping some WP locales to PayPal locales
+			if ( isset( $locale, $this->_locales_mapping ) ) {
```

Comment:
I'm assuming this was meant to be `isset( $this->_locales_mapping[ $locale ]; )` rather than `isset( $locale, $this->_locales_mapping )`?

Reply:
Looking at the latest commit I saw this recently changed from an `array_key_exists()` so this was just a carryover from that. With that in mind I decided I would just patch that.

Reply:
🤦‍♀️